src/content/docs/en/upgrade-astro.mdx

```diff
@@ -180,3 +180,22 @@ A major release may also include some non-breaking changes and improvements that
   Once these features are out of their experimental period, they will follow the normal semantic versioning contract.
 
 - **Improvements to the documentation** (e.g. reference and error messages). They are built from source for the `docs` repository. This allows Astro to quickly update docs fixes and improvements in the cases where documentation source content is stored in the main `astro` repository.
+
+### Node.js support and upgrade policies
+
+#### Support
+
+- Astro supports the **latest _extended_ LTS** versions of Node.js.
+- Astro can support odd versions of Node.js.
+
+#### Upgrade
+
+- Odd versions of Node.js can be deprecated and/or dropped when the next even version of Node.js published. This change can occur in a **minor** release of Astro, after an arbitrary period of time decided by the Astro Core team.
+- Upgrade of the minimum **_extended_ LTS** version of Node.js can occur in a **minor** release of Astro.
+  - If a security flaw in Node.js that **affects Astro** is disclosed and fixed, the Core team can bump the minimum version of the **_extended_ LTS** in a **patch** release.
+- Upgrade of minor or major versions of Node.js - **not** extended LTS - occurs only in major versions of Astro.
+  - If a security flaw in Node.js that **affects Astro** is disclosed and fixed, the Core team can bump the minimum version in a **minor** release.
+
+### Extended maintenance
+ 
+The Core team will provide extended maintenance e.g. security fixes, only to the previous major of the current release. This means that of the **current major** is `v4.*`, the Core team will backport security fixes on for `v3.*` versions. 
```

Comment:
Just pointing out that this implies that you will provide other maintenance too, and not JUST security fixes. (Not sure I've seen that actually happen? I *think* I've only seen older versions released for security reasons?)

If you really mean "just security fixes" then I would say that! Otherwise, this is pretty vague and makes it sound like v3 is *regularly maintained* too.

Reply:
That's fair, I'll update it. Only security fixes